### ISSUE TITLE
Add Grid/Computational Domain

### ISSUE THREAD

**Issue by rpachaly** [FLO-2DSoftware/FLO-2DMapCrafter#37]

**Comment by rpachaly** [comment]
Noemi note:

1. Easy way to display Grid cells for project review without loading the project. For example: when you click on the cell of the raster, I see the depth and coordinates but not the grid ID.

**Comment by FLO-2DKaren** [comment]
Add a grid to Site Characteristics.
It could be a raster but the ID is grid number.

<img width="922" height="690" alt="Image" src="https://github.com/user-attachments/assets/876f0e20-0a69-4f5d-a721-743988d4aadd" />

**Comment by CorneliusKibet** [comment]
@rpachaly, @FLO-2DKaren,

Hi all. I was thinking what if we re-recreate the actual grids instead of creating a raster? See the attached sample (I used the cell/grid id column of DEPTH.OUT file).

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/4cc14634-0309-4524-8a15-26ad9d2fd4a5" />

**Comment by FLO-2DKaren** [comment]
Yes we need an actual grid not a raster because it needs stuff like centroid, grid id and such.  I like it in Site Characteristics.

**Comment by FLO-2DKaren** [comment]
tested
